admin_server/src/main/java/com/backend/admin_server/access_requests/service/RequestOverrideService.java

```diff
@@ -0,0 +1,26 @@
+package com.backend.admin_server.access_requests.service;
+
+import com.backend.admin_server.access_requests.dto.AccessRequestDTO;
+import com.backend.admin_server.access_requests.model.AccessRequestModel;
+import com.backend.admin_server.access_requests.repository.AccessRequestRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class RequestOverrideService {
+
+    private final AccessRequestRepository accessRequestRepository;
+
+    public RequestOverrideService(AccessRequestRepository accessRequestRepository) {
+        this.accessRequestRepository = accessRequestRepository;
+    }
+
+    public AccessRequestModel updateRequestState(String requestId, AccessRequestDTO accessRequestDTO) {
+        AccessRequestModel accessRequestModel = accessRequestRepository.findByRequestId(requestId);
+
+        accessRequestModel.setApprovalStatus(accessRequestDTO.getApprovalStatus());
+        accessRequestModel.setState(accessRequestDTO.getState());
```

Comment:
Are the changes to the DTO in this PR? When I look I don't see it in the list. Nor do I see the state in the DTO currently.

Reply:
Yes, that was the previous PR